npm/css-src/sass/overrides/_base.scss

```diff
@@ -1,3 +1,7 @@
 html {
     box-sizing: border-box;
 }
+
+.hide {
```

Comment:
Should this be in the main SCSS files, rather than in the overrides folder? I assumed /overrides was for anything that overrides HTML defaults or styling in the libraries we use.

Reply:
Moved to main.scss